models/comments-models.js

```diff
@@ -9,4 +10,18 @@ exports.selectCommentsByArticleId = (articleId) => {
     .then((response) => {
         return response.rows
     });
-};
\ No newline at end of file
+};
+
+exports.insertComment = (body, username, article_id,) => {
+    const created_at = new Date(Date.now())
```

Comment:
does your DB not do this for you?

Reply:
I think it only does it for the initial seed data, as this is a new comment I though it'd have to be done in the model?